### PR TITLE
♻️ make call to docker-compose verbose to ease debugging

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
Since we do see strange behavior sometimes starting containers, this will help (in conjunction with setting the dy-sidecar log level to DEBUG)

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]

**Review by GitHK — COMMENTED** [review]
Could we maybe have verbose only on the calls that you actually need?
Not sure how nice this will work with the pull, when it's pulling a very big image.

**Comment by sanderegg** [comment]
> Could we maybe have verbose only on the calls that you actually need? Not sure how nice this will work with the pull, when it's pulling a very big image.

well, let's see what master says about it.
right now we have no idea of what is going on, and it's only in conjunction with DEBUG log level

**Comment by GitHK** [comment]
> > Could we maybe have verbose only on the calls that you actually need? Not sure how nice this will work with the pull, when it's pulling a very big image.
> 
> well, let's see what master says about it. right now we have no idea of what is going on, and it's only in conjunction with DEBUG log level

I'm fine with that but at this point only add the verbose flag if you select debug mode, this way we do not always have it enabled even at different log levels.

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ async def docker_compose_config(
     """
     result = await _write_file_and_spawn_process(
         compose_spec_yaml,
-        command='docker-compose --file "{file_path}" config',
+        command='docker-compose --verbose --file "{file_path}" config',
```
Comment (pcrespov) [inline services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py:70]: what about adding it as argument to the function?

```python

async def docker_compose_...(compose_spec_yaml: str, *, timeout: int=60, verbose: bool=False) -> CommandResult:
  options = []
  if verbose:
      options.append("--verbose")

  # ...
  command= f'docker-compose {" ".join(options)} --file "{{file_path}}" config',

```

This way this slowly converges into a meaningful API for docker-compose commands

**Review by pcrespov — APPROVED** [review]

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py (hunk 1 above)
Comment (sanderegg) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py]: ok I did a variation around this and @GitHK request.

**Review by GitHK — APPROVED** [review]
👍 Looks great, thanks!

**Pull request merged**